floodviz/static/js/floodviz_onready.js

```diff
@@ -0,0 +1,31 @@
+document.addEventListener("DOMContentLoaded", function (event) {
+
+	"use strict";
+
+	// Map figure setup
+	var map_height = FV.mapinfo.height;
```

Comment:
It seems unnecessary to put each conf variable in it's own var declaration and then create a var map_options. Also for map_options format it so each key: value pair is on its own line.